src/v/cluster/feature_table.h

```diff
@@ -70,12 +72,19 @@ struct feature_spec {
     prepare_policy prepare_rule;
 };
 
-constexpr static std::array feature_schema{feature_spec{
-  cluster_version{1},
-  "central_config",
-  feature::central_config,
-  feature_spec::available_policy::always,
-  feature_spec::prepare_policy::always}};
+constexpr static std::array feature_schema{
+  feature_spec{
+    cluster_version{1},
+    "central_config",
+    feature::central_config,
+    feature_spec::available_policy::always,
+    feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{2001},
+    "__test_alpha",
+    feature::test_alpha,
+    feature_spec::available_policy::explicit_only,
+    feature_spec::prepare_policy::always}};
```

Comment:
Another option might be to enable the test specs with a #ifdef and then define that ifdef on the cmake target for the test

Reply:
This is used in integration (ducktape) test, so needs to be compiled into the real redpanda binary.  If there's more stuff in future that's specifically for unit tests that can go in an ifdef though.

Reply:
Makes sense I didn't realize that it was for integration.